packages/sampler/stories/qa/Scroller.js

```diff
@@ -126,6 +127,34 @@ class ScrollerWithLargeContainer extends React.Component {
 	}
 }
 
+class ScrollerWithFocusInContainer extends React.Component {
+	constructor (props) {
+		super(props);
+	}
+
+	render = () => {
```

Comment:
Also, I think we should not be using arrow functions for `render`.  This is simple enough we can convert this into an SFC.  We don't need any state here.